common/service.sh

```diff
@@ -2,7 +2,6 @@
 # More info in the main Magisk thread
 
 if [ $API -ge 28 ] && [ $FULL ]; then
-  pm enable "com.google.android.apps.wellbeing/com.google.android.apps.wellbeing.autodnd.ui.AutoDndGesturesSettingsActivity"
```

Comment:
Figure out how to remove updates before installing WellBeing

Reply:
Are you referring to Aurora's method? Pretty sure @Stallix tested and it doesn't work.

I guess this line is unnecessary code then too.

If we did want to remove updates it would be a part of the pm --uninstall command or wiping /data/data and /data/app.